### DISCUSSION TITLE
Paging doesn't work for me

### DISCUSSION THREAD

**Discussion by caelia** [nushell/nushell#4216] [Q&A]
Hi! I'm pretty "nu" to Nushell, but I like what I'm seeing so far ... don't know yet if I want to use it all the time, but it seems like there are some very useful innovations.

Anyway, one thing that is important to me in a shell is the ability to page through long listings - e.g. directory listings. In `bash` or `zsh` I'd just go

    > ls /foo/bar |less 

It appears that Nushell has builtin paging capability, and as per the configuration section of the user guide, I put the following in my config file:

    [textview]
    paging_mode = "QuitIfOneScreen"
    pager = "less"

And I've confirmed, with `config get textview` that these settings are in effect when I run Nushell ... but paging does not happen. Have I missed some necessary parameter, or are there things I should check in the environment?

I'm running Nushell v0.41 in the Sakura terminal emulator on Manjaro Linux.

TIA for any tips!

**Comment by sophiajt** [comment]
Hi @caelia - thanks for the nice comments :)

We're currently working on some changes that will help you use external pagers like `less`. Currently, we do a lot of the drawing in nushell and handle the display ourselves. In an upcoming set of big changes to nushell, we're hoping to make this much more flexible so you can use a pager like `less` to look at tables.
